integrations/helm/release/release.go

```diff
@@ -254,16 +245,16 @@ func (r *Release) Install(chartPath, releaseName string, fhr flux_v1beta1.HelmRe
 
 		if err != nil {
 			r.logger.Log("error", fmt.Sprintf("Chart upgrade release failed: %s: %#v", fhr.Spec.ReleaseName, err))
-			return nil, err
+			return nil, ValuesChecksum(rawVals), err
```

Comment:
As described above, we may return an error here while the release has been moved forward.